README.md

```diff
@@ -113,28 +113,28 @@ def filter_genre(initial_dataset: pd.DataFrame, selected_genre):
 
 *This is the execution graph of the scenario we are implementing*
 
-<div align="center"><img src="readme_img/readme_exec_g.png" alt="Taipy Core Graph"  width="60%"/></div>
+<div align="center"><img src="readme_img/readme_exec_graph.png" alt="Taipy BE Graph"  width="40%"/></div>
 
 
 ### Taipy Studio - The easy peasy way
 *You can use the Taipy Studio extension in VSCode to configure your sequence with no code*
 
-<div align="center"><img src="readme_img/readme_studio.gif" width="80%" alt="GUI demo"></img></div>
+<div align="center"><img src="readme_img/readme_demo_studio.gif" width="60%" alt="demo BE"></img></div>
 
 *Your configuration is automatically saved as a TOML file*
 
 <br>
 <br>
 
 ### <div align="center">*Find out more*</div>
-*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-core/) and [Documentation](https://docs.taipy.io/en/latest/manuals/studio/) </div>*
+*<div align="center">Check out our [Documentation](https://docs.taipy.io/en/latest/manuals/studio/) </div>*
 
 <br>
 <br>
 <br>
 <br>
 
-### Taipy Core - a walk on the code side
+### Taipy Back-End - a walk on the code side
 <div align="left">For more advanced use cases or if you prefer coding your configurations instead of using Taipy Studio, Taipy has your back! </div>
 
 *<div align="left">Check out the movie genre demo scenario creation with this [Demo](https://www.taipy.io/project/movie-genre-selector/) </div>*
```

Comment:
Put the docs.taipy.io link and not the link of the website https://docs.taipy.io/en/latest/knowledge_base/demos/movie_genre_selector/